src/main/java/org/mitre/synthea/modules/LifecycleModule.java

```diff
@@ -249,10 +252,21 @@ public static void birth(Person person, long time) {
    * @param person The person to generate vital signs for.
    */
   private static void setupVitalSignGenerators(Person person) {
+    boolean usePhysiology = Boolean.parseBoolean(Config.get("physiology.enabled", "false"));
```

Comment:
Typically, we are setting these type of variables once from the Config file at the class level. See `appendNumbersToNames` in the same file.